tests/queries/window_state_by_two/query.trickle

```diff
@@ -0,0 +1,12 @@
+use tremor::origin;
+define window by_two from tumbling
+state
+  0
+script
```

Comment:
should this use the `script from tick`?

Reply:
no `tick` is for the periodic tick we use for timed windows, is is a group by two window that groups two events together it doesn't need ticks

Reply:
Considering this as an example it could do a `drop` in the `script from tick` branch? We'd also test that the `tick` port handling works.